static/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
This will conflict with our existing distribution file (where our current index/css files are). Can we delete it?